helmfile/values/networkpolicy-common.yaml.gotmpl

```
@@ -0,0 +1,46 @@
+global:
+  apiserver:
+  {{- if eq .Environment.Name "service_cluster" }}
+    ips: {{- toYaml .Values.networkPolicies.global.scApiserver.ips | nindent 6 }}
+    port: {{ .Values.networkPolicies.global.scApiserver.port }}
+  {{- else }}
+    ips: {{- toYaml .Values.networkPolicies.global.wcApiserver.ips | nindent 6 }}
+    port: {{ .Values.networkPolicies.global.wcApiserver.port }}
+  {{- end }}
+  nodes:
+  {{- if eq .Environment.Name "service_cluster" }}
+    ips: {{- toYaml .Values.networkPolicies.global.scNodes.ips | nindent 6 }}
+  {{- else }}
+    ips: {{- toYaml .Values.networkPolicies.global.wcNodes.ips | nindent 6 }}
+  {{- end }}
+  ingress:
+  {{- if eq .Environment.Name "service_cluster" }}
+    ips: {{- toYaml .Values.networkPolicies.global.scIngress.ips | nindent 6 }}
+  {{- else }}
+    ips: {{- toYaml .Values.networkPolicies.global.wcIngress.ips | nindent 6 }}
+  {{- end }}
+  objectStorage:
+    ips: {{- toYaml .Values.networkPolicies.global.objectStorage.ips | nindent 6 }}
+    ports: {{- toYaml .Values.networkPolicies.global.objectStorage.ports | nindent 6 }}
+  externalLoadBalancer: {{ .Values.networkPolicies.global.externalLoadBalancer }}
+  ingressUsingHostNetwork: {{ .Values.ingressNginx.controller.useHostPort }}
+
+kured:
+  enabled: {{ and .Values.kured.enabled .Values.networkPolicies.kured.enabled }}
+  notificationSlack:
+    enabled: {{ .Values.kured.notification.slack.enabled }}
+    ips: {{- toYaml .Values.networkPolicies.kured.notificationSlack.ips | nindent 6 }}
+    ports: {{- toYaml .Values.networkPolicies.kured.notificationSlack.ports | nindent 6 }}
+
+velero:
+  enabled: {{ and .Values.velero.enabled .Values.networkPolicies.velero.enabled }}
+
+certManager:
+  enabled: {{ .Values.networkPolicies.certManager.enabled }}
+  {{- if eq .Environment.Name "service_cluster" }}
+  namespaces: {{ toYaml .Values.networkPolicies.certManager.namespaces | nindent 4 }}
+  {{- else }}
+  namespaces: {{ toYaml .Values.user.namespaces | nindent 4 }}
+  {{- end }}
```

I think we should be able to add more namespaces to workload cluster as well.
```suggestion
  namespaces: {{ toYaml .Values.networkPolicies.certManager.namespaces | nindent 4 }}
  {{- if eq .Environment.Name "workload_cluster" }}
  namespaces: {{ toYaml .Values.user.namespaces | nindent 4 }}
  {{- end }}
```

---

And then add the config option to `wc-config.yaml` as well.